package.json

```diff
@@ -38,6 +38,7 @@
     "enzyme": "^2.5.0",
     "eslint": "^4.7.2",
     "eslint-config-stripes": "^0.0.1",
+    "html-webpack-plugin": "^2.30.1",
```

Comment:
FYI, The plugin will need to be included under "dependencies" rather than "devDependencies".  Dev dependencies won't get installed via npm/yarn when stripes-core is itself a dependency of clean and un-linked platform build.

Reply:
Moved dependency as you suggested.